README.md

```diff
@@ -93,20 +94,22 @@ Spline is written for mobile browsers. This means that Spline should support new
 
 Other browsers may be supported but Spline is not actively developed on them. If you find any issues with specific browsers, please log them in our [Issues tracker](http://github.com/mobify/spline/issues).
 
-## The Mobify CSS Triforce ![Triforce](http://img4.wikia.nocookie.net/__cb20091113050618/zelda/images/1/1a/Obtained_Triforce_Piece.gif)
-
-Spline is part of the _Mobify CSS Triforce_: a collection of three UI Development projects built for making mobile e-commerce builds easier.
+## How To Test
 
-- **Spline**: A mobile-first mixin and function library for Sass.
-- [**Vellum**](http://github.com/mobify/vellum): A set of default styles for mobile-first builds.
-- [**Stencil**](http://github.com/mobify/stencil): A library of common UI patterns for mobile e-commerce builds.
+1. Run `npm install`
+2. Run `npm run lint`
```

Comment:
No change necessary here (unless you wanna), but just as a "nice-to-know": For these numbered lists in markdown you can just keep all entries as "1." and they will be numbered in order as expected.

The benefit of doing it this way is that you can easily modify the list without manually editing the numbers into correct ordering.